jadx-gui/src/main/java/jadx/gui/utils/UiUtils.java

```diff
@@ -56,6 +46,22 @@
 	private UiUtils() {
 	}
 
+	public static SVGIcon openSvgIcon(String name) {
+		String iconPath = "/icons/" + name + ".svg";
+		InputStream in = UiUtils.class.getResourceAsStream(iconPath);
+		if (in == null) {
+			throw new JadxRuntimeException("Icon not found: " + iconPath);
+		}
+		BufferedReader reader = new BufferedReader(new InputStreamReader(in));
+		URI uri = SVGCache.getSVGUniverse().loadSVG(reader, iconPath);
+
+		SVGIcon svgIcon = new SVGIcon();
+		svgIcon.setAntiAlias(true);
+		svgIcon.setScaleToFit(true);
+		svgIcon.setSvgURI(uri);
+		return svgIcon;
```

Comment:
```suggestion
		String iconPath = "icons/" + name + ".svg";
		return new FlatSVGIcon(iconPath);
```
Using `FlatSVGIcon` fix icons brightness difference, now some of them looks like disabled.